src/sort.c

```diff
@@ -185,7 +185,7 @@ int sortCompare(const void *s1, const void *s2) {
     return server.sort_desc ? -cmp : cmp;
 }
 
-/* The SORT command is the most complex command in Redis. Warning: this code
+/* The SORT command is the server's most complex command. Warning: this code
```

Comment:
Eh, on second thought, I think keeping valkey here probably makes more sense.
```suggestion
/* The SORT command is Valkey's most complex command. Warning: this code
```

Reply:
done